src/Aspire.Hosting.Python/PythonProjectResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Python;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Python applications to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class PythonProjectResourceBuilderExtensions
+{
+    /// <summary>
```

Comment:
The `<summary>` should be a short one sentence thing. Move the rest into `<remarks>` (after `<returns>`).

Would be good to add an `<example>` as well for each of the extension methods.

Reply:
@wmeints just wondering if you've had a chance to come back and look at the <remarks> and <example> on this PR?